bitrise.yml

```diff
@@ -3,13 +3,18 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: ""
 app:
   envs:
-  - SAMPLE_APP_ANDROID: https://github.com/guness/Test-Android.git
+  - SAMPLE_APP_ANDROID: https://github.com/bitrise-io/sample-apps-android-sdk22.git
+  - SAMPLE_APP_IOS: https://github.com/bitrise-io/sample-apps-ios-simple-objc.git
   - BITRISE_STEP_ID: firebase-app-distribution
-  - BITRISE_STEP_VERSION: 0.0.1
+  - BITRISE_STEP_VERSION: 0.0.2
```

Comment:
No need to increment step version, before step is published.

Also, by convention, `0.1.0` should be used for initial StepLib PR -- and then use semver for each new release.

Reply:
haven't sent this test to release before merging. If you can you test and give green light, then I can merge and send new PR with version 0.1.0